bin/Cargo.toml

```diff
@@ -32,57 +32,24 @@ default = ["sqlite", "resolver", "native-certs", "ascii-art"]
 # if enabled, the hickory-dns binary will print ascii-art on start, disable to reduce the binary size
 ascii-art = []
 
-dnssec-openssl = [
-    "dnssec",
-    "hickory-client/dnssec-openssl",
-    "hickory-proto/dnssec-openssl",
-    "hickory-server/dnssec-openssl",
-]
-dnssec-ring = [
-    "dnssec",
-    "hickory-client/dnssec-ring",
-    "hickory-proto/dnssec-ring",
-    "hickory-server/dnssec-ring",
-]
+dnssec-openssl = ["dnssec", "hickory-server/dnssec-openssl"]
+dnssec-ring = ["dnssec", "hickory-server/dnssec-ring"]
 dnssec = []
 recursor = ["hickory-server/recursor"]
 # Recursive Resolution is Experimental!
```

Comment:
Is this comment meant to go on the `recursor` feature?

Reply:
Yes, it is (pre-existing issue). Want to send a PR?

Reply:
It should go on this PR, otherwise this looks good